docs/docs-content/clusters/edge/upgrade-behavior.md

```diff
@@ -0,0 +1,79 @@
+| No-op                | Acknowledges the update request but does nothing.                                                                                                       |
+
+## Upgrade Behavior By Change Location
+
+Updates in the OS layer or the Kubernetes lay can trigger different upgrade behavior depending on the exact values
+changed.
+
+### OS Layer
+
+| Repave                          | Reboot                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | Service Restart | No-op                                                                                                                                                                                                                                                                                                       |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <li> `options.system.uri` </li> | <li>`stages.rootfs.*`</li> <li>`stages.initramfs.*`</li> <li>`stages.boot.*`</li> <li>`stages.fs.*`</li> <li>`stages.network.*`</li> <li>`stages.after-install.*`</li> <li>`stages.after-install-chroot.*`</li> <li>`stages.after-upgrade.*`</li> <li>`stages.after-upgrade-chroot.*`</li> <li>`stages.after-reset.*`</li> <li>`stages.after-reset-chroot.*`</li> <li>`stages.before-install.*`</li> <li>`stages.before-upgrade.*`</li> <li>`stages.before-reset.*`</li> | None.           | <li>`pack.*`</li> <li>`providerCredentials.*`</li> <li>`options.system.registry`</li> <li>`options.system.repo`</li> <li>`options.system.k8sDistribution`</li> <li>`options.system.osName`</li> <li>`options.system.peVersion`</li> <li>`options.system.customTag`</li> <li>`options.system.osVersion`</li> |
+
+### Kubernetes Layer
```

Comment:
Do we have the same values for the three tabs? Just to confirm :)

Reply:
I noticed that too.

Reply:
yep, they are all the same. The content was provided to me in three tabs, and i made them separate so that in the future if they differ, it's easy to manage